recipe/activate.sh

```diff
@@ -0,0 +1 @@
+export TRITON_LIBDEVICE_PATH=${CONDA_PREFIX}/lib/libdevice.10.bc
```

Comment:
I would really like to avoid this environment variable. It becomes very fragile when working with nested environments.

While nested environments arent something that we need to support, maybe there is a different way that would make this less fragile.

What of we only looked at ${CONDA_PReFiX} instead? It seems that in the python that is what is done, while in the C code, you've opted for the use of a dedicated environment variable.

Reply:
Done in the latest series of commits - I agree with your point.